pkg/client/packet/client.go

```diff
@@ -0,0 +1,33 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
If you don't need this client let's completely remove it. Or will you use it for something?

Reply:
Hmm... it is used by `pkg/operation/cloudbotanist/packetbotanist/packetbotanist.go`. 

Oh, wait. You are saying that the botanist only uses it to interact directly with the cloud provider API (as well as generate the charts that get passed to the seed). If there is no need to interact directly (e.g. get security groups or the like), then we don't need the client?

I get that, but I can see use cases where the botanist would interact directly with the provider. There are things that we are not supporting in here yet like L2 networking, new capabilities, etc. This already works, even if it doesn't actually call anything. I would like to leave it in, making future integrations much simpler.

Reply:
> Oh, wait. You are saying that the botanist only uses it to interact directly with the cloud provider API (as well as generate the charts that get passed to the seed). If there is no need to interact directly (e.g. get security groups or the like), then we don't need the client?

Yes, exactly!

> I get that, but I can see use cases where the botanist would interact directly with the provider. There are things that we are not supporting in here yet like L2 networking, new capabilities, etc. This already works, even if it doesn't actually call anything. I would like to leave it in, making future integrations much simpler.

OK, anyway, this will move out with extensibility, so fine for me :) I was just wondering why it exists in the first place because nothing is implemented on that client.